src/Module.php

```diff
@@ -412,4 +433,18 @@ protected function defaultVersion()
         }
         return parent::defaultVersion();
     }
+
+    /**
+     * @return string
+     */
+    public function htmlTitle(){
+        if(is_array($this->html_title)){
+            $url = Url::base(true);
+            if(array_key_exists($url, $this->html_title))
+                return $this->html_title[$url];
+        }
+        elseif(!empty($this->html_title))
+            return $this->html_title;
+        return $this->default_html_title;
+    }
```

Comment:
I think it would be better if you used a single property to handle this:

```
    /**
     * @var mixed the debugger page title, either a string or an array where keys are the base url or the
     * current request. Defaults to "Yii Debugger".
     *
     * ```php
     * 'pageTitle'=> [
     *     'base-url'=> 'page title',
     *     'http://qa.example.test'=> 'qa debugger'
     * ]
     * ```
     */
   $pageTitle = 'Yii Debugger';   
```

and use a hardcoded `return "Yii Debugger";` if an array was passed and it doesn't contain the current request base url.

Could you also apply [Yii coding style](https://github.com/yiisoft/yii2/blob/master/docs/internals/core-code-style.md) to your code ?